packages/components/infinite-scroll/src/index.ts

```diff
@@ -162,6 +162,7 @@ const InfiniteScroll: ObjectDirective<
     container.addEventListener('scroll', onScroll)
   },
   unmounted(el) {
```

Comment:
```suggestion
  beforeUnmount(el) {
```
@btea What do you think?

Reply:
OK